ROS/osr/src/roboclaw_wrapper.py

```diff
@@ -221,7 +240,7 @@ def drive_cmd_cb(self, cmd):
         props = self.roboclaw_mapping["drive_right_front"]
         vel_cmd = self.velocity2qpps(cmd.right_front_vel, props["ticks_per_rev"], props["gear_ratio"])
         self.send_velocity_cmd(props["address"], props["channel"], vel_cmd)
-
+        """
```

Comment:
Don't comment out code like this. Commenting out obsolete code is almost never useful and it makes reviewing harder since we don't see what code becomes inactive.